pymc3/tests/test_distributions_random.py

```diff
@@ -1497,7 +1494,6 @@ def ref_rand(size, mu, sigma):
 
         pymc3_random(pm.Moyal, {"mu": R, "sigma": Rplus}, ref_rand=ref_rand)
 
-    @pytest.mark.xfail(reason="This distribution has not been refactored for v4")
     @pytest.mark.xfail(condition=(aesara.config.floatX == "float32"), reason="Fails on float32")
     def test_interpolated(self):
```

Comment:
This one should be moved to it's own function. The reason for this is that after refactoring all distributions the `TestScalarParameterSamples` where this one is will disappear, but this distribution is special enough that it should be kept (i.e., we are not simply drawing from scipy / numpy distributions).

Reply:
I just moved it to it's own class and named it `TestInterpolated`